### PR TITLE
button, cta, rating color updates

### PR DESCRIPTION
**Pull request by cowills**
## Description

Updating button, CTA, and everything that isn't an SVG in rating.

- icon-only button with border will be handled as a separate ticket
- need new SVG assets from design in order to update the rating stars

### Cross-browser testing:
- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari
- [x] IE11
- [x] iOS
- [x] Android

### Visual regression testing:
- [x] Passes with existing reference images (or failed as expected)

### A11y:
- [x] Meets WCAG AA standards

### Documentation:
- [x] API docs created/updated

### CODE REVIEW

File: src/components/button/styles/CdrButton.scss [hunk 1]
```diff
@@ -1,26 +1,3 @@
-/* ==========================================================================
```
Comment (mhewson) [inline src/components/button/styles/CdrButton.scss:1]: generally I think the idea of a TOC is something to be preserved, this should be an artifact that helps ensure the completeness of the styles contained within. 
what reasons would we have to remove this?

**Review by mhewson — DISMISSED** [review]

File: src/components/button/styles/CdrButton.scss (hunk 1 above)
Comment (cowills) [reply src/components/button/styles/CdrButton.scss:1]: we don't keep them up to date. not all of our CSS files have TOCs. you can't actually navigate from the TOC directly to a style definition. our style files should be small anyways. 

happy to discuss it at dev critique if anyone feels strongly about keeping them

Comment (mhewson) [reply src/components/button/styles/CdrButton.scss:1]: Yea lets talk about what we could do in the future

**Review by mhewson — APPROVED** [review]

File: src/components/button/styles/CdrButton.scss (hunk 1 above)
Comment (cowills) [reply src/components/button/styles/CdrButton.scss:1]: adding a card to dev critique :+1:

**Pull request merged**